apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -279,15 +293,40 @@ if ( tracksUserData ) {
 	initializeWithIdentity( tracksUserData );
 }
 
-registerPlugin( 'page-templates', {
+// Don't open plugin if we arent creating new page.
+// Is there better way?  Can we acess get_current_screen for wordpress screen object like in php?
+if ( window.location.toString().includes( 'post-new' ) ) {
```

Comment:
I would like to find a better way to tell if we are going into a new post to load the template selector by default.
I have tried both `select( 'core/editor' ).isEditedPostNew()` and `select( 'core/editor' ).isCleanNewPost()` but they both initially load as false in this context.